app/code/Magento/Ui/Component/AbstractComponent.php

```diff
@@ -141,8 +142,6 @@ protected function prepareChildComponent(UiComponentInterface $component)
 
     /**
      * Produce and return block's html output
-     *
-     * @return string
```

Comment:
According to the interface `UiComponentInterface` and method `render` this method must return string. I think here we need to return the result instead of changing PHPDocs

Reply:
ok